app/app.go

```diff
@@ -196,6 +196,11 @@ var (
 		emissionsModuleTypes.UndistributedObserverRewardsPool: nil,
 		emissionsModuleTypes.UndistributedTssRewardsPool:      nil,
 	}
+disa
+	// module accounts that are allowed to receive tokens
+	blockedReceivingModAcc = map[string]bool{
```

Comment:
Might be a good idea to iterate over all module accounts and set them all to false by default , and we can turn on for the ones that w need to allow to receive funds 
`app.ModuleAccountAddrs()`